src/__tests__/cogniteClient.spec.ts

```diff
@@ -72,6 +73,43 @@ describe('CogniteClient', () => {
     });
   });
 
+  describe('http requests', () => {
+    let mock: MockAdapter;
+    let client: CogniteClient;
+
+    beforeAll(async () => {
+      client = setupClient();
+      mock = new MockAdapter(client.instance);
+    });
+    beforeEach(async () => {
+      mock.reset();
+    });
+
+    test('get method', async () => {
+      mock.onGet('/').reply(200, 'test');
```

Comment:
Should rather use `replyOnce`. Same for all other tests

Reply:
Done